config/locales/en/activerecord/course/assessment/answer.yml

```diff
@@ -2,19 +2,20 @@ en:
   activerecord:
     attributes:
       course/assessment/answer:
-        submitted_at: "Submitted At"
-        grade: "Grade"
-        grader: "Grader"
-        graded_at: "Graded At"
+        submitted_at: 'Submitted At'
+        grade: 'Grade'
+        grader: 'Grader'
+        graded_at: 'Graded At'
     errors:
       models:
         course/assessment/answer:
           attributes:
             question:
-              consistent_assessment: "must belong to the same assessment"
+              consistent_assessment: 'must belong to the same assessment'
             submission:
-              attemptable_state: "must be in the attempting state"
+              attemptable_state: 'must be in the attempting state'
+              newly_updated: 'the version to be saved is not the most updated one'
```

Comment:
Should this message be improved?. If i am a student reading this, would I know what does this mean and what should i do?